templates/definition/meter/sungrow-hybrid.yaml

```diff
@@ -165,15 +153,39 @@ render: |
               type: writesingle
               decode: uint16
         - source: const
-          value: 0xAA
+          value: 0xAA # Charge
           set:
             source: modbus
             {{- include "modbus" . | indent 10 }}
             timeout: {{ .timeout }}
             register:
-              address: 13050 # Forced mode
+              address: 13050 # Charge/discharge command
               type: writesingle
               decode: uint16
+        - source: go
+          in:
+          - name: power
+            type: int
+            config:
+              source: modbus
+              {{- include "modbus" . | indent 12 }}
+              timeout: {{ .timeout }}
+              register:
+                address: 5627 # BDC rated power
```

Comment:
@premultiply sicher, dass das hier nicht 5628 sein muss wie in der doku?

Reply:
Bei der Sungrow-Doku muss man immer aufpassen denn die haben da ein +1-Offset drin.
Damit es ein bisschen schwieriger wird beim Lesen...

Reply:
Bei `13050` machen wir das aber nicht.

Reply:
`13051` ist im Template auch ohne Offset. Also so wie im PDF.

Reply:
Da müsst ihr euch bitte einigen. Vmtl klärt es sich wenn man ins Log schaut ;)

Reply:
Ich bestehe nicht darauf dass es so wie derzeit richtig sein muss. 😇
Wie gesagt, ich komme da selber ständig durcheinander. Also kann auch gut sein dass ich mich vertan hab.